src/startDebugging.ts

```diff
@@ -7,10 +7,12 @@ import {
 import * as vscode from 'vscode';
 import { getConfigValue } from './utils';
 import { forgeBuildTask, foundryRoot, loadBuildInfo } from './foundry';
+import { WorkspaceWatcher } from './WorkspaceWatcher';
 
 export async function startDebugging(
   contract: ContractDefinition,
-  method: FunctionDefinition
+  method: FunctionDefinition,
+  worksapceWatcher: WorkspaceWatcher
```

Comment:
There's a typo here.
```suggestion
  workspaceWatcher: WorkspaceWatcher
```

Reply:
Nice catch. Fixed in [1753732](https://github.com/runtimeverification/simbolik-vscode/pull/30/commits/17537328b5d453c2a3a5c1138a255730ea2eb77a)